features/b.yml

```
@@ -1,7 +1,6 @@
-draft_date: 2024-06-05
 name: <b>
-description: The `<b>` element TODO.
+description: The `<b>` HTML element makes text bold in order to draw the reader's attention to it.
```

Optionally, we could align all of these to use the same pattern, like this:

```suggestion
description: The `<b>` HTML element draws reader's attention to text, styling text as bold by default.
```